pkg/cache/informer_cache_unit_test.go

```diff
@@ -75,4 +76,11 @@ var _ = Describe("ip.objectTypeForListObject", func() {
 			Expect(obj).To(Equal(referenceObject))
 		})
 	})
+
+	It("should set Limit field", func() {
```

Comment:
We need to test behavior, not implementation details, i.E. here that it returns at most `Limit` objects even if the cache has more

Reply:
Oh I see. Sorry about that. To be sure, such a test would reside in `cache_test.go`?

Reply:
In pkg/cache, the file name is not particularly relevant